DTDL/options_persistence.json

```diff
@@ -0,0 +1,60 @@
+{
+  "@context": "dtmi:dtdl:context;2",
+  "@id": "dtmi:io:tributech:options:persistence;1",
+  "@type": "Interface",
+  "extends": "dtmi:io:tributech:options:base;1",
+  "displayName": "Persistence options",
+  "contents": [
+    {
+      "@type": "Property",
+      "name": "PersistenceProvider",
+      "schema": {
+        "@type": "Enum",
+        "valueSchema": "string",
+        "enumValues": [
+          {
+            "name": "postgre",
+            "displayName": "PostgreSQL",
+            "enumValue": "POSTGRE"
+          },
+          {
+            "name": "influx",
+            "displayName": "InfluxDB",
+            "enumValue": "INFLUX"
+          }
+        ]
+      },
+      "writable": true,
+      "displayName": "Data API persistence provider",
+      "description": "Select a persistence provider for the stream."
+    },
+    {
+      "@type": "Property",
+      "name": "PersistType",
```

Comment:
@tributech-dhofer @mayrmax should we align it with `encoding` in `base_stream.json`?

Reply:
As discussed via call with @mayrmax its ok for now.